lib/src/SortableList/SortableItem.tsx

```diff
@@ -0,0 +1,28 @@
+import React from "react";
+import classnames from "classnames";
+
+export type SortableItemProps = Omit<JSX.IntrinsicElements["div"], "onDragStart" | "onDragOver" | "onDragEnd"> & {
+    uniqueKey: string;
+    disabled?: boolean;
+};
+
+const SortableItem: React.FC<SortableItemProps> = React.forwardRef(
+    ({ className, disabled, children, uniqueKey, ...props }: React.PropsWithChildren<SortableItemProps>, ref: React.ForwardedRef<HTMLDivElement>) => {
```

Comment:
`uniqueKey` is not being used, can we remove them?

Reply:
this uniquekey will be used for reference when in SortableList component